BE/sidedish11/src/main/java/dev/codesquad/java/sidedish11/exception/DataNotFoundException.java

```diff
@@ -0,0 +1,15 @@
+package dev.codesquad.java.sidedish11.exception;
+
+public class DataNotFoundException extends RuntimeException{
+    private final int ERROR_CODE;
```

Comment:
static 변수가 아니므로 upper camel을 사용할 필욘 없습니다. 

```suggestion
    private final int errorCode;